guides/tfvc-to-git.md

```diff
@@ -65,5 +65,7 @@ This allows you to version your documentation with the code it's documenting.
 Taking it a step further to change your documentation to markdown is even better.
 That will allow you to track changes through your documentation, and allow you to begin to automate the documentation.
 
-Wiki's should be used to document parts of your application that don't (are very unlikely to) change from version to version.
+Wiki's should be used to document client or user information while documenation related to development or deployments should be kept with the source code itself.
```

Comment:
```suggestion
Wikis should be used to document client or user information while documentation related to development or deployments should be kept with the source code itself.
```

"Wikis" seems to be the plural used by Wikipedia. [link](https://en.wikipedia.org/wiki/List_of_wikis)